digitalearthau/paths.py

```diff
@@ -9,10 +9,12 @@
 
 import pathlib
 import structlog
+import logging
 
 from datacube.utils import is_supported_document_type, read_documents, InvalidDocException, uri_to_local_path
 
 _LOG = structlog.getLogger()
+_LOG1 = logging.getLogger(__file__)
```

Comment:
I don't think it makes sense to log in both plain text and structured forms. It'd be better to stay consistent with what is here and use the `structlog`.

Reply:
Ok. I will modify the code to be consistent with structured form.